<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>661</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="table_screen">
     <widget class="QPushButton" name="navigate_add_table">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>590</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 25px;
border: none;</string>
      </property>
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
     <widget class="QPushButton" name="navigate_czw">
      <property name="geometry">
       <rect>
        <x>273</x>
        <y>1</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
color: black;
border-top-style: none;
border-right-style: solid;
border-left-style: solid;
border-bottom-style: solid;
border-bottom-width: 2px;
border-right-width: 2px;
border-bottom-color: black;
border-left-color: black;
border-right-color: black;
border-bottom-right-radius:5px;
border-bottom-left-radius:5px;
</string>
      </property>
      <property name="text">
       <string>Czwartek</string>
      </property>
     </widget>
     <widget class="QPushButton" name="navigate_pt">
      <property name="geometry">
       <rect>
        <x>367</x>
        <y>1</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
color: black;
border-top-style: none;
border-right-style: solid;
border-left-style: solid;
border-bottom-style: solid;
border-bottom-width: 2px;
border-right-width: 2px;
border-bottom-color: black;
border-left-color: black;
border-right-color: black;
border-bottom-right-radius:5px;
border-bottom-left-radius:5px;
</string>
      </property>
      <property name="text">
       <string>Piątek</string>
      </property>
     </widget>
     <widget class="QPushButton" name="navigate_sb">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>1</y>
        <width>101</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
color: black;
border-top-style: none;
border-right-style: solid;
border-left-style: solid;
border-bottom-style: solid;
border-bottom-width: 2px;
border-right-width: 2px;
border-bottom-color: black;
border-left-color: black;
border-right-color: black;
border-bottom-right-radius:5px;
border-bottom-left-radius:5px;
</string>
      </property>
      <property name="text">
       <string>Sobota</string>
      </property>
     </widget>
     <widget class="QPushButton" name="navigate_niedz">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>1</y>
        <width>101</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
color: black;
border-top-style: none;
border-right-style: none;
border-left-style: solid;
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: black;
border-left-color: black;
border-bottom-left-radius:5px;
</string>
      </property>
      <property name="text">
       <string>Niedziela</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_plus_table">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>610</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(icons/down.png);
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="start_minus_table">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(icons/up.png);;
border:none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="navigate_wt">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>1</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
color: black;
border-top-style: none;
border-right-style: solid;
border-left-style: solid;
border-bottom-style: solid;
border-bottom-width: 2px;
border-right-width: 2px;
border-bottom-color: black;
border-left-color: black;
border-right-color: black;
border-bottom-right-radius:5px;
border-bottom-left-radius:5px;
</string>
      </property>
      <property name="text">
       <string>Wtorek</string>
      </property>
     </widget>
     <widget class="QPushButton" name="navigate_sr">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>1</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
color: black;
border-top-style: none;
border-right-style: solid;
border-left-style: solid;
border-bottom-style: solid;
border-bottom-width: 2px;
border-right-width: 2px;
border-bottom-color: black;
border-left-color: black;
border-right-color: black;
border-bottom-right-radius:5px;
border-bottom-left-radius:5px;
</string>
      </property>
      <property name="text">
       <string>Środa</string>
      </property>
     </widget>
     <widget class="QPushButton" name="navigate_pon">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>1</y>
        <width>101</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
color: black;
border-top-style: none;
border-right-style: solid;
border-left-style: none;
border-bottom-style: solid;
border-bottom-width: 2px;
border-right-width: 2px;
border-bottom-color: black;
border-left-color: black;
border-right-color: black;
border-bottom-right-radius:5px;
</string>
      </property>
      <property name="text">
       <string>Poniedzialek</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="task_screen">
     <widget class="QPushButton" name="navigate_add_tasks">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>590</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 25px;
border: none;</string>
      </property>
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_minus_task">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(icons/up.png);
border:none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="start_plus_task">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>610</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(icons/down.png);
border:none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="insert_to_tasks">
     <widget class="QTextEdit" name="description_tasks">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px dotted black;
background-color: rgb(192,192,192);
font-size: 20px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_tasks">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>271</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-left-style: none;
border-right-style: none;
border-top-style: none;
border-bottom-style: dotted;
border-bottom-width: 2px;
background-color: rgb(192,192,192);
font-size: 20px;</string>
      </property>
      <property name="maxLength">
       <number>29</number>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_tasks">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-left-style: none;
border-right-style: none;
border-top-style: none;
border-bottom-style: dotted;
border-bottom-width: 2px;
background-color: rgb(192,192,192);
color: balck;
font-size: 20px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
color: black;</string>
      </property>
      <property name="text">
       <string>Nazwa:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
color: black;</string>
      </property>
      <property name="text">
       <string>Opis:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>175</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
color: black;</string>
      </property>
      <property name="text">
       <string>Termin:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_to_tasks">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 25px;
border: none;</string>
      </property>
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_task">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>290</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 25px;
border: none;</string>
      </property>
      <property name="text">
       <string>Edytuj</string>
      </property>
     </widget>
     <widget class="QComboBox" name="color_task">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>180</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-left-style: none;
border-right-style: none;
border-top-style: none;
border-bottom-style: dotted;
border-bottom-width: 2px;
background-color: rgb(192,192,192);
color: balck;
font-size: 20px;</string>
      </property>
      <item>
       <property name="text">
        <string>Niebieski</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fioletowy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pomarańczowy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Zielony</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Czerwony</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Żółty</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
color: black;</string>
      </property>
      <property name="text">
       <string>Kolor tła:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="insert_to_notes">
     <widget class="QTextEdit" name="description_notes">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>341</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,204);
border: 3px dotted black;
font-size:20px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 23px;
color: balck;</string>
      </property>
      <property name="text">
       <string>Treść notatki:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_to_notes">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>280</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 25px;
border: none;</string>
      </property>
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_note">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>340</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 25px;
border: none;</string>
      </property>
      <property name="text">
       <string>Edytuj</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="notes_screen">
     <widget class="QPushButton" name="navigate_add_notes">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>590</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 25px;
border: none;</string>
      </property>
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_minus_note">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>335</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(icons/left.png);
border:none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="start_plus_note">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>335</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(icons/right.png);
border:none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="insert_to_table">
     <widget class="QLineEdit" name="name_table">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>271</width>
        <height>28</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>271</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-left-style: none;
border-right-style: none;
border-top-style: none;
border-bottom-style: dotted;
border-bottom-width: 2px;
background-color: rgb(192,192,192);
font-size: 20px;</string>
      </property>
      <property name="maxLength">
       <number>29</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="description_table">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px dotted black;
background-color: rgb(192,192,192);
font-size: 20px;</string>
      </property>
      <property name="maxLength">
       <number>44</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
color: black;</string>
      </property>
      <property name="text">
       <string>Nazwa: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
color: black;</string>
      </property>
      <property name="text">
       <string>Opis: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_to_table">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 25px;
border: none;</string>
      </property>
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
color: black;</string>
      </property>
      <property name="text">
       <string>Godzina:</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="date_table">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>118</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-left-style: none;
border-right-style: none;
border-top-style: none;
border-bottom-style: dotted;
border-bottom-width: 2px;
background-color: rgb(192,192,192);
color: balck;
font-size: 20px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_table">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>290</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 25px;
border: none;</string>
      </property>
      <property name="text">
       <string>Edytuj</string>
      </property>
     </widget>
     <widget class="QComboBox" name="color_table">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>180</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-left-style: none;
border-right-style: none;
border-top-style: none;
border-bottom-style: dotted;
border-bottom-width: 2px;
background-color: rgb(192,192,192);
color: balck;
font-size: 20px;</string>
      </property>
      <item>
       <property name="text">
        <string>Niebieski</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fioletowy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pomarańczowy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Żółty</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Czerwony</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Zielony</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>175</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
color: black;</string>
      </property>
      <property name="text">
       <string>Kolor tła:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="navigate_table">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>221</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
font-style: italic;
color: black;
border-top-style: none;
border-right-style: solid;
border-left-style: solid;
border-bottom-style: solid;
border-bottom-width: 2px;
border-left-width: 2px;
border-right-width: 2px;
border-bottom-color: black;
border-left-color: black;
border-right-color: black;
</string>
    </property>
    <property name="text">
     <string>Plan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="navigate_tasks">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>221</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
font-style: italic;
color: black;
border-top-style: none;
border-right-style: solid;
border-left-style: solid;
border-bottom-style: solid;
border-bottom-width: 2px;
border-left-width: 2px;
border-right-width: 2px;
border-bottom-color: black;
border-left-color: black;
border-right-color: black;
</string>
    </property>
    <property name="text">
     <string>Zadania</string>
    </property>
   </widget>
   <widget class="QPushButton" name="navigate_notes">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>221</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
font-style: italic;
color: black;
border-top-style: none;
border-right-style: solid;
border-left-style: solid;
border-bottom-style: solid;
border-bottom-width: 2px;
border-left-width: 2px;
border-right-width: 2px;
border-bottom-color: black;
border-left-color: black;
border-right-color: black;
</string>
    </property>
    <property name="text">
     <string>Notatki</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
